README.md

```diff
@@ -0,0 +1,11 @@
+# Digital Assets Directory #
```

Comment:
@antoineherzog / @BenSim, is more required in the README?  If so then please add it.

Reply:
good with me